src/config.ts

```diff
@@ -87,7 +87,9 @@ const DefaultConfig: Partial<Libp2pInit> = {
     maxInboundStreams: 1,
     maxOutboundStreams: 1,
     maxPushIncomingStreams: 1,
-    maxPushOutgoingStreams: 1
+    maxPushOutgoingStreams: 1,
+    disableIdentify: false,
+    disableIdentifyPush: false
```

Comment:
Do we need this flag? Is there a scenario where we would disable identify push but not identify?

Reply:
yes, in lodestar we probably want to disable identify-push but keep identify